### DISCUSSION TITLE
Specifying a control member in gen-ens-prod and ensemble-stat

### DISCUSSION THREAD

**Discussion by michelleharrold** [dtcenter/METplus#1595] [Tips/Tricks]
For a DTC project to optimize ensemble design in the Rapid Refresh Forecast System, we are in the midst of setting up the necessary verification and wanted to get some feedback on the pros/cons of providing a control member in gen-ens-prod and ensemble-stat. I briefly talked with @JohnHalleyGotway about this newer capability in MET, and while it was a good conversation, he suggested reaching out the community via Discussions to get feedback (specifically, he mentioned @mpm-meto from the UK Met Office since they are specifying a control member in their verification).

I know from the MET User Guide that the control member is included in the computation of the ensemble mean but then excluded from the spread calculation. But I am trying to ascertain the exact benefits and the statistical impact?

Thank you for any input(s) regarding this!
Michelle Harrold

**Comment by JohnHalleyGotway** [comment]
@michelleharrold this is a great topic for discussion. Thanks for getting it started. FYI, I'm going to move it over from the [Statistical Computation](https://github.com/dtcenter/METplus/discussions/categories/statistical-computation) category into the [Tips/Tricks](https://github.com/dtcenter/METplus/discussions/categories/tips-tricks) category. The latter is the only category we currently have for open discussion topics. All the other categories are question/answer format. I expect that @mpm-meto will offer some perspectives on the benefits of defining a control member. But I think this should remain an open discussion rather us choosing a specific "Answer" and locking future conversation on it.

**Comment by michelleharrold** [comment]
Thanks @JohnHalleyGotway for finding the right home for this topic! @mpm-meto - if you have any tips on the best path forward with respect to defining a control member, it would be much appreciated!
